src/app/timer/timer.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="timer">
+    <div class="timer__row">
+        <p class="timer__time timer__time--hours">{{this.date.getHours()}} <span class="timer__time-span">Hour(s)</span></p>
+        <p class="timer__time timer__time--minuts">{{this.date.getMinutes()}} <span class="timer__time-span">minuts</span></p>
+        <p class="timer__time timer__time--seconds">{{this.date.getSeconds()}} <span class="timer__time-span">seconds</span></p>
```

Comment:
usually stopwatch looks like 00:00:00, so let's try to implement the same structure